packages/table-dev-app/src/index.scss

```
@@ -144,6 +144,10 @@ body {
   background: $light-gray5;
 }
 
+.#{$ns}-dark .tbl-zebra-stripe {
+  background-color: $dark-gray5;
+}
+
```

```suggestion
  background: $light-gray5;

  .#{$ns}-dark & {
    background-color: $dark-gray5;
  }
}
```

Thoughts on this small refactor to more clearly tie the light mode and dark mode colors together? I think it more closely matches how dark mode is applied throughout the rest of the file as well.

---

Agree! 3702f861627f504046bff7c50a42d7bbd271dccb